pkgs/top-level/python-packages.nix

```diff
@@ -16225,8 +16225,6 @@ self: super: with self; {
 
   unstructured = callPackage ../development/python-modules/unstructured { };
 
-  unstructured-client = callPackage ../development/python-modules/unstructured-client { };
```

Comment:
This is in the wrong commit.

Reply:
yes, my bad, do you mind if we leave it as is, I'm probably going to mess it up again.